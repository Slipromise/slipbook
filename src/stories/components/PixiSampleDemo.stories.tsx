import { Sprite, Stage } from "@pixi/react";
import { Meta, StoryObj } from "@storybook/react";
import image from "../assets/gift.png"

const meta: Meta<typeof Sprite> = {
  component: Sprite,
  parameters: {
      layout: "centered",
  },
};

export default meta;

type Story = StoryObj<typeof Sprite>;

// TODO: PixiStoryDecorator

export const Normal: Story = {
  args: {
    image,
    x: 0,
    y: 0,
  },
  render: function Render(args) {
    return (
      <Stage width={1024} height={1024} options={{ background: 0x1099bb }}>
        <Sprite {...args} />
      </Stage>
    );
  },
};
