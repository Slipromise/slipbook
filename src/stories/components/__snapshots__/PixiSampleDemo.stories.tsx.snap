// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stories/components/PixiSampleDemo Normal smoke-test 1`] = `
<canvas width="400"
        height="300"
        style="width: 400px; height: 300px;"
>
</canvas>
`;
